### PR TITLE
N-QUEEN.cpp

### PR DESCRIPTION
**Pull request by thakur24**
Added N-QUEEN.cpp file providing input , source code and output of the same. It is one of the popular questions of Backtracking.

### CODE REVIEW

File: CPP/Algorithm/LCA.cpp [hunk 1]
```diff
@@ -1,112 +0,0 @@
-/*input
```
Comment (lowjack1) [inline CPP/Algorithm/LCA.cpp:1]: Is there any issue with this file? why did you delete this file? You didn't mention the reason also you didn't send any replacement for the same

File: CPP/Algorithm/N-QUEEN.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
+Input: The chess board, the col where the queen is trying to be placed.
+
```
Comment (lowjack1) [inline CPP/Algorithm/N-QUEEN.cpp:2]: Please follow the existing code standard. You didn't comment out the statements which is not the part of C++. 
It looks like you have copy pasted it from tutorials points because I have found exact similar code here [N-Queen Problem](https://www.tutorialspoint.com/N-Queen-Problem).

**Review by lowjack1 — CHANGES_REQUESTED** [review]

File: CPP/Algorithm/N-QUEEN.cpp (hunk 2 above)
Comment (thakur24) [reply CPP/Algorithm/N-QUEEN.cpp:2]: Let me review it once and i will fix it .
